shell/platform/embedder/embedder.h

```diff
@@ -562,6 +588,157 @@ typedef struct {
   const FlutterTaskRunnerDescription* platform_task_runner;
 } FlutterCustomTaskRunners;
 
+typedef struct {
+  // The type of the OpenGL backing store. Currently, it can either be a texture
+  // or a framebuffer.
+  FlutterOpenGLTargetType type;
+  union {
+    // A texture for Flutter to render into.
+    FlutterOpenGLTexture texture;
+    // A framebuffer for Flutter to render into. The embedder must ensure that
+    // the framebuffer is complete.
+    FlutterOpenGLFramebuffer framebuffer;
+  };
+} FlutterOpenGLBackingStore;
+
+typedef struct {
```

Comment:
Should this have a size field, or are you 100% confident that it won't grow over time?

Reply:
I was trying to make this consistent with the `FlutterOpenGLTexture` and `FlutterOpenGLFramebuffer` structs. No, I am not 100% confident.